src/Hr/index.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withTheme } from '../Theme';
+import Container from '../Container';
+import View from '../View';
+import Text from '../Text';
+
+const Hr = (props) => {
+  const {
+    text,
+    color,
+    thickness,
+    marginTop,
+    marginBottom,
+    shadow,
+  } = props;
+
+  let textShadowStyle;
+  let shadowStyle;
+
+  if (shadow) {
+    textShadowStyle = {
+      textShadowColor: shadow.color,
+      textShadowOffset: { width: shadow.offset.width, height: shadow.offset.height },
+      textShadowRadius: shadow.radius,
+    };
+
+    shadowStyle = {
+      shadowColor: shadow.color,
+      shadowOffset: { width: shadow.offset.width, height: shadow.offset.height },
+      shadowRadius: shadow.radius,
+    };
+  }
+
+  return (
+    <Container style={
+      {
+        flexDirection: 'row',
+        justifyContent: 'center',
+        marginTop,
+        marginBottom,
+      }
+    }
+    >
+      <View style={
+        {
+          flex: 0.5,
+          backgroundColor: color,
+          height: thickness,
+          width: '100%',
+          alignSelf: 'center',
+          ...shadowStyle,
+        }
+      }
+      />
+      <Text type={color} auto style={{ color, ...textShadowStyle }}>
+      &nbsp;
+        { text }
+      &nbsp;
+      </Text>
+      <View style={
```

Comment:
I think we should allow to use the Hr component without text. You can do something like
 
```
{text === null ? (
  <Text ...>
    {...}
  </Text>
) : null}
```

Reply:
Makes sense. I'll look into that now.

Reply:
This is done check on commit 89af9e75bb70ac069cbd3067d7681efad11c52a4